snapshot/README.md

```diff
@@ -174,6 +174,10 @@ fastlane snapshot
 
 Your screenshots will be stored in the `./screenshots/` folder by default (or `./fastlane/screenshots` if you're using [fastlane](https://fastlane.tools))
 
+New with Xcode 9, *snapshot* can run multiple simulators concurrently. This is the default behavior in order to take your screenshots as quickly as possible. This can be disabled to run each device, one at a time, by setting the `:concurrent_simulators` option to `false`.
+
+**Note:** While running *snapshot* with Xcode 9, the simulators will not be visibly spawned. So, while you wont see the simulators running your tests, they will, in fact, be taking your screenshots.
```

Comment:
Is that something we also tell the user about in the snapshot output?

Reply:
I dont think we do mention that in the output. We probably should though! 👍

Reply:
Cool, is that something you're gonna submit a follow-up PR for?

Reply:
Agreed, I was confused by this today.